labgrid/driver/sshdriver.py

```diff
@@ -103,24 +105,24 @@ def _check_master(self):
         return self._start_own_master()
 
     @Driver.check_active
-    @step(args=['cmd'], result=True)
-    def run(self, cmd, codec="utf-8", decodeerrors="strict", timeout=None): # pylint: disable=unused-argument
-        return self._run(cmd, codec=codec, decodeerrors=decodeerrors)
+    @step(args=['command'], result=True)
+    def run(self, command: str, *, timeout: int = 30):
```

Comment:
Why do we want to have a default timeout for the SSHDriver? Compared to the serial based drivers, SSH easily detects a crashed target, so introducing such a short timeout might be surprising.

Reply:
I thought it would be a good thing to have consistent timeouts over all drivers implementing the `CommandProtocol`. I understand now that timeouts are a way to detect a crashed target in case of serial communication. In that case I will remove the default timeout from the `CommandProtocol` and set it to None for the `SSHDriver`, alright?

Reply:
Implemented as suggested, see below.